src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Js/doctypegrideditor.controllers.js

```diff
@@ -206,8 +206,16 @@ angular.module("umbraco").controller("Our.Umbraco.DocTypeGridEditor.Dialogs.DocT
 
             function loadNode() {
                 contentResource.getScaffold(-20, $scope.model.dialogData.docTypeAlias).then(function (data) {
-                    // Remove the last tab
-                    data.tabs.pop();
+
```

Comment:
i think this will do inside the existing loop..  whatcha think?

if (!data.tabs[t].active) { continue; }

Reply:
This didn't work for me.

Reply:
@markglibres what loop are you talking about

@meregan what exactly does not work for you ? Because i see a report of the fix working here : https://our.umbraco.org/projects/backoffice-extensions/doc-type-grid-editor/doc-type-grid-editor-feedback/90527-dtge-dialog-editor-empty-in-umbraco-781

Reply:
@markglibres sorry removing   data.tabs.pop(); works the get current umbraco version code from @dawoe didn't work for me.

Reply:
@dawoe the loop after it pops the last tab. i just noticed that umbraco api returns active=false for the properties tab on older versions, and no info tab on the latest release.